MantidPlot/src/ProjectSaveView.cpp

```diff
@@ -265,6 +265,10 @@ void ProjectSaveView::save(bool checked) {
   auto filePath = m_ui.projectPath->text();
   auto compress = filePath.endsWith(".gz");
 
+  m_ui.btnSave->setEnabled(false);
```

Comment:
Is the dialog guaranteed to close after the save so the buttons will never be needed again?

Reply:
@ConorMFinn Could I get clarification on this?

Reply:
I don't believe so, if the file cannot be saved another dialog opens that closes the window once okay is clicked.

Reply:
I don't believe so, if the file cannot be saved another dialog opens that closes the window once okay is clicked.

Reply:
Thanks for clarifying.